### PR TITLE
E2e metadata empty transforms

### PR DESCRIPTION
**Pull request by Droogans**

### CODE REVIEW

File: src/rxMetadata/rxMetadata.page.js [hunk 1]
```diff
@@ -25,6 +25,17 @@ var rxMetadata = {
         }
     },
 
+    terms: {
+        get: function () {
+            return this.rootElement.$$('div.label').map(function (keyElem) {
+                return keyElem.getText().then(function (key) {
+                    // strip ending colon character
+                    return key.slice(0, -1);
```
Comment (Droogans) [inline src/rxMetadata/rxMetadata.page.js]: @CITguy will metadata *always* have a colon at the end? Is it configurable?

Comment (CITguy) [inline src/rxMetadata/rxMetadata.page.js]: Yes, https://github.com/rackerlabs/encore-ui/blob/master/src/rxMetadata/templates/rxMeta.html#L2

Comment (CITguy) [inline src/rxMetadata/rxMetadata.page.js]: I believe this was a design decision, but I could be wrong. @glynnis, @ElementE

**Comment by CITguy** [comment]
@Droogans, can you please rebase your branch. There seem to be duplicate changes from #1130

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Comment by TheNando** [comment]
LGTM

**Pull request merged**